src/runtime/table-of-contents.ts

```diff
@@ -41,3 +42,24 @@ onContentReady(() => {
 
     tableOfContents(toc, headings);
 });
+
+const mediaQueryLarge = window.matchMedia(`(min-width: ${BREAKPOINT_LARGE})`);
+mediaQueryLarge.addEventListener("change", toggleTableOfContents);
+toggleTableOfContents(mediaQueryLarge);
```

Comment:
Är detta för att den ska köras initialt?

Reply:
Yes. Behövdes för att sätta initial state när man laddar in sidan. Sen i `navigation.ts` för att sätta initial state när man kör replace på innehållet.